app/views/users/index.html.erb

```diff
@@ -1,15 +1,6 @@
-<h1>Users!</h1>
-Our blog has <%= User.count %> Users and counting!
-<ul>
+<%= render 'layouts/title', title: "Our Blog App has #{User.count} Users and counting!" %>
+<div class="user user-shelf">
   <% @users.each do |user| %>
-    <li>
-      <strong>User name : </strong>
-      <%= link_to  user.name, user %>
-      <% if !user.posts_counter %>
-        <p><%= user.name %> has not Posted 📪 anything Yet </p>
-      <% else %>
-        <p>Post count <strong><%= user.posts_counter%></strong></p>
-      <% end %>
-    </li>
+    <%= render 'user_card', user: user %>
   <% end %>
```

Comment:
✅ Good job using partials here.

Reply:
Hi @TadesseDev  

I am happy to see you take a deep look, I want to say that the app mock is not dead, it is still under construction i want to add more styling on `Home page` like part of bio should be there for User card

Reply:
I see, It's great. But we are required to make sure the readme is free from broken links.

Reply:
Well, can you tell me which changes you are talking about in `optional` changes  😁😁

as far now 
I can see only this 👇👇
![image](https://github.com/PowerLevel9000/Blog/assets/116451144/87159791-ff88-40ea-801d-b0d22f211aa1)

Reply:
ya, about that. The `users_controler` class methods implementations. the ` @users = User.all` and `` @user = User.find(params[:id])` this are part of this milestone. I hope that make sense @PowerLevel9000

Reply:
![image](https://github.com/PowerLevel9000/Blog/assets/116451144/6295efc4-5bbe-4649-9389-26f9137f864e)

Ahh I understand, I implemented it in my previous `feature branch` next time I will note it